templates/zerver/integrations.html

```diff
@@ -1830,6 +1830,47 @@
       <img class="screenshot" src="/static/images/integrations/stash/005.png" />
     </div>
 
+    <div id="stripe" class = "integration-instructions">
+
+      <p>You can choose to be notified whenever certain events are triggered on Stripe
+      by adding a webhook to your Stripe account. First, create a stream where the events
+      are to be sent.</p>
+
+      <p>Next, on your {{ settings_html|safe }}, create a Stripe bot.</p>
+
+      <p>Add a webhook to your Stripe account by going to "Your account"->"Account settings"->"Webhooks"->"Add Endpoint"
+         and adding the following URL, replacing the bot email address(with the <code>@</code>sign escaped to <code>%40</code>), bot API key, Zulip stream, and Zulip topic with the appropriate
+         values: </p>
+      <p><code>{{ external_uri_scheme }}<font color="#00A26F">stripe-bot%40example.com</font>:<font color="#00A26F">api_key</font>@{{ external_api_path_subdomain }}/v1/external/stripe?stream=<font color="#00A26F">sales</font>%26topic=<font color="#00A26F">Stripe</font></code></p>
```

Comment:
Could you break it into separate lines? It's hard to read. Generally, try to have lines of about 80 characters, e.g. the first paragraph of this document is well done.